go.mod

```diff
@@ -40,4 +40,5 @@ require (
 	gopkg.in/check.v1 v1.0.0-20190902080502-41f04d3bba15 // indirect
 	gopkg.in/yaml.v2 v2.2.8 // indirect
 	gorm.io/driver/sqlite v1.1.3 // indirect
+	gorm.io/gorm v1.20.1
```

Comment:
Will this introduce a conflict with jinzhu/gorm?

Reply:
Should be okay.